src/test/java/uk/gov/justice/digital/delius/service/AppointmentServiceTest.java

```diff
@@ -34,17 +62,81 @@
     @BeforeEach
     @SuppressWarnings("unchecked")
```

Comment:
This warning is in the appointmentsSortedByContactDateDescending method - does the suppression need to move?

Reply:
Not needed